### PR TITLE
MON-3531 #comment new commit with cleaned files

### PR DESCRIPTION
**Pull request by EdytaSki**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -46,6 +42,8 @@ export { default as MessageError} from './Message/MessageError';
 export { default as MessageInfo} from './Message/MessageInfo';
 export { default as MessageStatus } from "./Message/MessageStatus";
 export { default as Navigation } from "./Navigation";
+export { default as NativeSelects } from "./FormMui";
+export { default as PageHdModifiers } from "./PageHdModifiers";
```
Comment (kduret) [inline src/index.js]: Did you test it ?
Cause I guess it will break cause those components are not on master branch...

Comment (EdytaSki) [reply src/index.js]: yes you are right - I will delete this.
I tested in my local with the hd-modifiers dev so that's why I didn't notice. Thanx.

**Comment by bdauria** [comment]
@EdytaSki What's the status here?

**Comment by bdauria** [comment]
@EdytaSki up

**Pull request closed without merging**